src/features/companyOverview/redux/reducer.test.ts

```diff
@@ -0,0 +1,57 @@
+import { CompanyOverview } from "../models"
+import { companyOverview } from "./reducer"
+
+const initalState: CompanyOverview = {
+    symbol: null,
+    companyName: null,
+    website: null,
+    description: null,
+    tags: [],
+}
+
+describe('Company reducer', () => {
+
+    it('should return the initial state', () => {
+        expect(companyOverview(
+            undefined, 
+            {} as any))
```

Comment:
why u need as any here?

Reply:
the as any was to cast the action as an any object on initial state to allow for testing of just the initial state return

This was implemented by Kyle :)